examples/demo-rollup/src/eth.rs

```diff
@@ -23,22 +22,14 @@ fn read_sov_tx_signer_priv_key() -> Result<DefaultPrivateKey, anyhow::Error> {
     Ok(key_and_address.private_key)
 }
 
-// TODO: #840
```

Comment:
Does this PR resolve https://github.com/Sovereign-Labs/sovereign-sdk/issues/840 ?